### PR TITLE
add concepts for train-sim v3

### PR DESCRIPTION
**Pull request by Castavo**
part of #205

### CODE REVIEW

File: content/docs/reference/design-docs/train-sim-v3/_index.en.md [hunk 1]
```diff
@@ -81,6 +67,62 @@ TrainSim([train simulator])
...
+Driving instructions are generated using domain constraints such as:
+
+- unsignaled line speed limits
+- permanent signaled speed limits
+- temporary speed limits
+- dynamic signaling:
+  - block / moving block
+  - dynamically signaled speed restrictions
+- neutral zones
+- stops
```
Comment (emersion) [inline content/docs/reference/design-docs/train-sim-v3/_index.en.md:99]: Ref https://github.com/OpenRailAssociation/osrd-website/pull/208 which touches this part as well

File: content/docs/reference/design-docs/train-sim-v3/_index.en.md [hunk 2]
```diff
@@ -81,6 +67,62 @@ TrainSim([train simulator])
...
+
+### Path properties
+
+Path properties are the physical properties of the path, namely elevation and curves.
```
Comment (emersion) [inline content/docs/reference/design-docs/train-sim-v3/_index.en.md:103]: Is electrification part of this as well?

File: content/docs/reference/design-docs/train-sim-v3/_index.en.md [hunk 3]
```diff
@@ -81,6 +67,62 @@ TrainSim([train simulator])
...
+
+### Driver behavior module
+
+The driver behavior module updates the train state based on:
```
Comment (emersion) [inline content/docs/reference/design-docs/train-sim-v3/_index.en.md:107]: To me, the main point of the driver behavior module is to represent both actual real-world driver behavior and margins. Maybe would be nice to give an example of these (e.g. driver anticipates a speed limit?), and state that both are represented with the same abstraction.

I'm not super sure we should get into the details of the inputs/outputs of the driver behavior module here in the high-level definition?

(Take this with a grain of salt, I'm no expert~)

Comment (Castavo) [reply content/docs/reference/design-docs/train-sim-v3/_index.en.md:107]: I feel like dicussing and pinning what the dreiver bahvior module represents would help us a lot, going forward

**Pull request merged**